docker-compose.yml

```diff
@@ -0,0 +1,89 @@
+version: '2'
+services:
+  web:
+    build:
+      context: ./resources/docker/web
+      args:
+        DAEMON_UID: "82"
+        DAEMON_GID: "82"
+    ports:
+      - "8080:8080"
+    volumes:
+      - .:/var/www/html
+    environment:
+      XDEBUG_CONFIG: "remote_enable=1 remote_connect_back=1 remote_host=10.254.254.254 remote_port=9000 idekey=PHPSTORM remote_autostart=0"
+      DOCUMENT_ROOT: "/var/www/html/web"
+      PHP_XDEBUG_ENABLED: "1"
+      PHP_IDE_CONFIG: "serverName=Docker"
+      PHP_SENDMAIL_PATH: "/usr/sbin/sendmail -t -i -S mailhog:1025"
+
+  # This will compile the CSS once when docker-compose is started.
+  sass:
+    image: pablofelix/sass
+    volumes:
+      - $PWD:/app/
+    entrypoint:
+      - watch
```

Comment:
Sass is not working, it goes down and it doesn't compile anything.
Also in the `compile-sass` step:
```
My Drupal Project > compile-sass:

     [sass] sass not found. Install sass or leafo scssphp.
```

Reply:
This is still there.

Reply:
As discussed in the chat with Francesco, this is caused by the `failonerror="false"` part of the command. It takes a bit of effort and a custom phing target to check if an executable exists in the system so we decided to go without it and simply not fail on error. Note that the file-exists condition cannot be used here as the path might not be absolute.
We can, of course, have a follow up to properly fix that.